package.json

```diff
@@ -14,7 +14,6 @@
   "scripts": {
     "build": "webpack",
     "build:development": "NODE_ENV=development webpack",
-    "build:staging": "NODE_ENV=staging webpack",
```

Comment:
Hrm... 

I can see us wanting to be able to do a no_mangle as an uglify option in staging some day...